index.src.html

```diff
@@ -761,18 +722,34 @@ <h4 id="algorithm-request" algorithm>Request a `Credential`</h4>
 
     5.  Let |origin| be the [=current settings object=]'s [=environment settings object/origin=].
 
-    6.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
-        ancestors=], and `false` otherwise.
+    6.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with
+        its ancestors=], and `false` otherwise.
```

Comment:
Is webauthn just going to ignore this flag, and rely on FP to block access to requests instead?

Reply:
webauthn uses this flag for a couple of things:
1. webauthn's create() checks `sameOriginWithAncestors` and returns an error if cross-origin, because a permissions policy (nee Feature Policy) is presently instantiated only for webauthn's get()(i.e., [[DiscoverFromExternalSource]]()).
2. whether or not a webauthn get() operation occurred cross-origin is recorded in webauthn's `CollectedClientData`.